src/Toaster/Toaster.js

```diff
@@ -9,43 +9,103 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.​
 
-import React, { Component, Fragment } from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 import { toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 
+import { StyledToaster } from './Toaster-styled';
+
+import {
+  StyledAlertContent,
+  StyledAlertIcon,
+  StyledAlertMessage
+} from '../Alert/Alert-styled';
+
+import LightbulbIcon from 'calcite-ui-icons-react/LightbulbIcon';
+import CheckCircleIcon from 'calcite-ui-icons-react/CheckCircleIcon';
+import ExclamationMarkTriangleIcon from 'calcite-ui-icons-react/ExclamationMarkTriangleIcon';
+
+const getAlertIcon = type => {
+  let defaultIcon;
+  if (type === 'success') {
+    defaultIcon = <CheckCircleIcon filled size={16} />;
+  } else if (type === 'warning' || type === 'error') {
+    defaultIcon = <ExclamationMarkTriangleIcon filled size={16} />;
+  } else {
+    defaultIcon = <LightbulbIcon filled size={16} />;
+  }
+
+  return <StyledAlertIcon>{defaultIcon}</StyledAlertIcon>;
+};
+
+// Wraps the content in a <ToastMessage /> component for styling
+// if they only passed a string of content
+const wrapContentMessage = content => {
+  if (content && typeof content === 'string') {
+    return <StyledAlertMessage>{content}</StyledAlertMessage>;
+  }
+
+  return content;
+};
+
+const notify = (
+  content,
+  { type, showProgress, showIcon, toastId, ...other } = {}
+) => {
+  let progressClassName = '';
+  if (showProgress) {
+    progressClassName = 'progress-visible';
+  } else if (showProgress === false) {
+    progressClassName = 'progress-hidden';
+  }
```

Comment:
Ternary here instead?

Reply:
There's essentially 3 options here, how would you propose writing it as a ternary?![bitmoji](https://render.bitstrips.com/v2/cpanel/35115333-7f99-467c-9705-d3ae6929374a-570f11fa-4425-4637-8c9d-ea0d0c2f1510-v1.png?transparent=1&palette=1&width=246)

Reply:
Ah, you're right, nvm